### ISSUE TITLE
Update OpenFEC New Relic integration to include legal resources search

### ISSUE THREAD

**Issue by porta-antiporta** [18F/fec-eregs#128]
To achieve re-authorization for beta.fec when we launch legal resources we need to ensure that we have appropriate monitoring/error reporting on the new search functionality. (Note there is a separate story that covers the fec-eregs app #130).

- [x] discuss with @LindsayYoung whether we also need to update the Sentry integration
- [x] Add new search URLs to New Relic monitoring suite
- [x] Trigger an error and confirm we are getting the reports and traces
- [ ] Update any system, incident-response plan documents (if needed)

**Comment by anthonygarvan** [comment]
Waiting on production deploy that will set WRITE_AUTHORIZED_TOKENS so that I can set up the ping to prod (already set on dev / stage / feature, failure tested on feature). @porta-antiporta where is the documentation that might need to be updated? Should I update you or?

**Comment by porta-antiporta** [comment]
@anthonygarvan Sorry `(if needed)` should really have been `(if it exists)`.  I already added a link from New Relic to fec-eregs in the latest change  I made to the beta.fec system diagram.

@LindsayYoung Do we currently have any incident response docs for beta.fec?

**Comment by LindsayYoung** [comment]
We don't have incident response docs specific to the project. The IT training we had, told us to stop the instance- don't delete it and make sure you preserve all logs etc.
